api/libproject/gcloudcontext.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "../../libproject/utils/common"
+require "json"
+
+class GcloudContextV2
+  attr_reader :account, :project, :env_key
+
+  def initialize(env_key)
+    @env_key = env_key
+    # We use both gcloud and gsutil commands for various tasks. While gcloud can take arguments, 
+    # gsutil uses the current gcloud config, so we just ensure a reasonable config is present here
+    # rather than allowing arguments.
```

Comment:
Reviewers take note.

Reply:
There's a reason we explicitly require --account and (particularly) --project arguments whenever messing with cloud resources... we want to make sure you don't accidentally update the wrong environment (e.g. update prod when you were trying to update test or whatever.)

Account doesn't change from env to env normally, so I don't care as much about that; but I definitely want to make --project required.